src/user/service.js

```diff
@@ -0,0 +1,54 @@
+import User from './User'
+
+const SORTING_VALUES = {
+  asc: 1,
+  desc: -1,
+}
+
+class UserService {
```

Comment:
I think it would be better not to disable eslint and just make these methods static
```suggestion
public static async create(user) {
}
```